src/extension.ts

```diff
@@ -72,9 +73,24 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(disposable);
 
-    disposable = vscode.commands.registerCommand('extension.ccUpdateDir', () => {
+    disposable = vscode.commands.registerCommand('extension.ccUpdateView', () => {
         if ( vscode.workspace.rootPath )
-            updatePath(vscode.workspace.rootPath);
+            updateObject(vscode.workspace.rootPath);
```

Comment:
Ah, ok. So we can leave the path empty. ```cleartool update``` will always update the hole snapshot view, no matter where you call it within the view.